src/parts/TypePart.php

```diff
@@ -59,19 +60,38 @@ private function parseType(Map $data) {
 		$this->collectionFormat = $data->get('collectionFormat');
 		$this->default = $data->get('default');
 		$this->maximum = $data->get('maximum');
-		$this->exclusiveMaximum = $data->has('exclusiveMaximum') && $data->get('exclusiveMaximum');
+		$this->exclusiveMaximum = $data->get('exclusiveMaximum');
```

Comment:
You should probably do something like:

```
$this->exclusiveMaximum = $data->get('exclusiveMaximum', false);
```

anyway the default value would be `null` and as such not `bool`.